src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -348,7 +348,9 @@ public function getContainer()
      * @param string $name      The cache name prefix
      * @param string $extension File extension of the resulting file
      *
-     * @deprecated since version 3.3, to be removed in 4.0.
+     * @deprecated The "loadClassCache()" method is deprecated since version 3.3 
```

Comment:
To be consistent with the other messages, it should probably something along the lines of: "since version 3.3, to be removed in 4.0. The class cache is not needed anymore when using PHP 7.0."